### PR TITLE
Added RatInMaze.py

### PR DESCRIPTION
**Pull request by Pranav016**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added rat-in-maze problem in python.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#1475

### CODE REVIEW

**Comment by yashaswibiyahut** [comment]
@Pranav016  kindly resolve the merge conflict firstly.

File: Python/backtracking/RatInMaze.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# 4
+# 1 1 1 1
+# 1 1 0 1
+# 1 1 1 1 
+# 1 1 1 1
+
+# Sample Output-
+# All possible paths will be printed
+
+# Time complexity- O(2^(n^2))
```
Comment (charlie219) [inline Python/backtracking/RatInMaze.py]: Use multi-lined comment instead of single-lined.
Include the actual Output for the Sample Input

**Review by charlie219 — CHANGES_REQUESTED** [review]

File: Python/backtracking/RatInMaze.py (hunk 1 above)
Comment (Pranav016) [reply Python/backtracking/RatInMaze.py]: Done

**Comment by Pranav016** [comment]
Can you please review this? @iamrajiv @atarax665 @shraddhavp
Thanks

**Comment by shraddhavp** [comment]
LGTM ! but please format it using http://pep8online.com/

**Comment by Pranav016** [comment]
> LGTM ! but please format it using http://pep8online.com/

Done 😄

**Pull request merged**